pkg/compose/watch.go

```diff
@@ -574,3 +585,114 @@ func (s *composeService) pruneDanglingImagesOnRebuild(ctx context.Context, proje
 		}
 	}
 }
+
+func (s *composeService) initialSync(ctx context.Context, project *types.Project, service types.ServiceConfig, trigger types.Trigger, ignore watch.PathMatcher, syncer sync.Syncer) error {
+	dockerFileIgnore, err := watch.NewDockerPatternMatcher("/", []string{"Dockerfile", "*compose*.y*ml"})
```

Comment:
is there a better way to handle the `compose.yaml` pattern? For instance, tried `compose.y[a]?ml` but it did not work :(